ui/hooks/useGasFeeInputs.js

```diff
@@ -164,6 +179,39 @@ export function useGasFeeInputs(defaultEstimateToUse = 'medium') {
     estimatedGasFeeTimeBounds,
   } = useGasFeeEstimates();
 
+  // This hook keeps track of a few pieces of transitional state. It is
+  // transitional because it is only used to modify a transaction in the
+  // metamask (background) state tree.
+  const [maxFeePerGas, setMaxFeePerGas] = useState(
+    transaction?.txParams?.maxFeePerGas
```

Comment:
So, in Swaps when we want to edit gas on the `View Quote` page, we don't have a transaction yet.

However, we store custom gasLimit, maxFeePerGas and maxPriorityFeePerGas in swapsState, which we could pass to the `EditGasPopover` component, e.g. in the `customGas` object instead of the `transaction` object. 

What do you think?

Reply:
@darkwing was this addressed ?

Reply:
@adonesky1 Yup, but in another place. @danjm has a commit that handles prefilling of the gas editing modal in Swaps for EIP-1559. We just need to get this PR in and then I can include Dan's commit in my PR that I will open shortly.

Reply:
Swaps can be addressed separately from this PR, IMO